task/sast-snyk-check/0.1/sast-snyk-check.yaml

```diff
@@ -9,16 +9,16 @@ metadata:
   name: sast-snyk-check
 spec:
   description: >-
-    Static code security test with snyk
+    Static code security test with snyk.
   results:
-    - description: Test output
+    - description: Tekton task test output.
       name: HACBS_TEST_OUTPUT
   params:
     - name: SNYK_SECRET
       default: ""
     - name: ARGS
       type: string
-      description: extra args needs to append
+      description: Extra args needs to append.
```

Comment:
does this mean that the user needs to append more arguments?

Reply:
It means some possible extra args can be appended.

Reply:
ok, what about: Append arguments.

if the issue is that there aren't enough args, then --> Append more arguments.

if it's mandatory, then --> Must append arguments.

(and add "more" after "append" if it's mandatory AND there aren't enough.)